main.py

```
@@ -11,43 +14,139 @@
+            if len(self.multi_leaderboard) > 10:
+                self.multi_leaderboard.pop()
+
+    def get_info(self):
+        info_string = f"Balance: {self.balance:,.6f} {CURRENCY_UNIT}\n"
+        info_string += "Rounds played: " + str(self.rounds_played) + "\n"
+        info_string += f"Total wagered: {self.total_wagered:,.6f} {CURRENCY_UNIT}\n"
+        info_string += f"Total win: {self.total_win:,.6f} {CURRENCY_UNIT}\n"
+        if self.rounds_played >= 5:
+            info_string += f"RTP: {(self.total_win/self.total_wagered)*100:06.2f} % / 096.53 %\n"
+        else:
+            info_string += f"RTP: ???.?? % / 096.53 %\n"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)